presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -23,6 +23,39 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+
+Connection security
+------------------
+
+The JDBC driver and connector automatically use Transport Layer Security (TLS) encryption and certificate validation. This requires a suitable TLS certificate configured on your SQL Server database host.
+
+To disable encryption in the connection string, use the encrypt property:
```

Comment:
```suggestion
To disable encryption in the connection string, use the `encrypt` property:
```
Let's help the user by formatting the name of the property in doc to emphasize what they need to use, and to help them find which part of the following example they should pay attention to.